doc/dev/release.md

```diff
@@ -306,11 +307,24 @@ OPERATORSUM="$(sha256sum operator-sdk | cut -d ' ' -f 1)"
 brew bump-formula-pr --strict --url=$OPERATORSDKURL --sha256=$OPERATORSUM operator-sdk
 ```
 
-Note: If there were any changes made to the CLI commands, make sure to look at the existing tests, in case they need updating.
+**Note**  If there were any changes made to the CLI commands, make sure to look at the existing tests, in case they need updating.
 
 You've now fully released a new version of the Operator SDK. Good work! Make sure to follow the post-release steps below.
 
-### (Post-release) Updating the operator-sdk-samples repo
+## Post-release steps
+
+Each of the following steps should have an associated JIRA ticket assigned to either yourself or another team member.
+
+### 1. Update the Upgrade version guide document and Go Memcached Sample
+
+1. The [Upgrade version guide][version-upgrade-guide] should be updated with the steps required for the users are able to migrate their project. Then, in order to know the steps and ensure that they will work upgrade the [Go Memcached Sample][go-memcached-sample].
```

Comment:
This section is still in the post-release steps. All the information needed to create this upgrade guide section is available when we make a release, so I don't see why this shouldn't be a release step. Move this to its own section in the release steps.

Reply:
Also still need to separate out the version upgrade guide updates from the samples updates. IIRC, we can't do the samples updates until after the release, because the samples require the new tag and base images.

Reply:
Done